forecast_solar/models.py

```diff
@@ -50,7 +56,9 @@ def power_production_now(self) -> int:
     @property
     def power_highest_peak_time_today(self) -> datetime:
         """Return datetime with highest power production moment today."""
-        now = datetime.now(tz=timezone.utc).replace(minute=59, second=59)
+        now = datetime.now(tz=zoneinfo.ZoneInfo(self.api_timezone)).replace(
+            minute=59, second=59
+        ) + timedelta(hours=24)
```

Comment:
Should we add 24 hours here for the today peak?

Reply:
This is only used to get the `day` value of tomorrow.

Reply:
Why is it called `today` then?

Reply:
oh shit that's a copy paste bug. Good catch!